### DISCUSSION TITLE
AIR for iOS: a loads b, and b loads c. Does that work?

### DISCUSSION THREAD

**Discussion by zhstar** [airsdk/Adobe-Runtime-Support#2514] [Q&A]
AIR for iOS: a loads b, and b loads c. Does that work?

**Comment by ajwfrost** [comment]
Assuming all of those are SWFs that have been processed ahead of time, to pull out their ActionScript and compile it into the initial application (i.e. 'b' and 'c' are primarily containing resources) then yes that should be possible....

But on iOS you always face the restriction that you can't just load in a SWF file that's containing ActionScript code, Apple don't allow this.
